dotCMS/src/main/java/com/dotcms/rest/api/v2/contenttype/FieldResource.java

```diff
@@ -184,13 +178,10 @@ public Response getContentTypeFieldByVar(@PathParam("typeId") final String typeI
             @PathParam("fieldVar") final String fieldVar, @Context final HttpServletRequest req)
             throws DotDataException, DotSecurityException {
 
```

Comment:
same here

Reply:
unused

Reply:
but I think this validation was removed https://github.com/dotCMS/core/blob/e152e08cd494d979dddb7a261286c5976a822d8f/dotCMS/src/main/java/com/dotcms/rest/WebResource.java#L453-L454